src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newState = { ...state };
```

Comment:
The initial cloning of `state` into `newState` is correct. This ensures that the original state is not mutated.